tests/components/tracked-controls.test.js

```diff
@@ -258,10 +257,7 @@ suite('tracked-controls', function () {
         call => call.args[0] === 'buttonchanged');
       assert.equal(emitChangedCalls.length, 1);
 
-      const emitCall = emitChangedCalls[0];
-      assert.equal(emitCall.args[0], 'buttonchanged');
-      assert.deepEqual(emitCall.args[1].id, 0);
-      assert.deepEqual(emitCall.args[1].state, controller.buttons[0]);
+      assertButtonEvent(emitChangedCalls[0], 'buttonchanged', 0, controller.buttons[0]);
```

Comment:
Helper is at the bottom. Basically avoids us writing a 4 lines of assert code and consolidates into a useful single call.

Reply:
cool